core/entity/v1/data/status.go

```diff
@@ -0,0 +1,18 @@
+/*
+|    Protect your secrets, protect your sensitive data.
+:    Explore VMware Secrets Manager docs at https://vsecm.com/
+</
+<>/  keep your secrets... secret
+>/
+<>/' Copyright 2023-present VMware Secrets Manager contributors.
+>/'  SPDX-License-Identifier: BSD-2-Clause
+*/
+
+package data
+
+type InitStatus string
```

Comment:
Renamed `KeystoneInitStatus` to `InitStatus`.

No need to tie it to the object. It indicates the init status of the system, not Keystone.